attestation-transformer/src/schemas.rs

```diff
@@ -81,14 +83,16 @@ impl FollowSchema {
 		r_bytes.copy_from_slice(&bytes[..32]);
 		s_bytes.copy_from_slice(&bytes[32..]);
 
-		FollowSchema { id: did, is_trustworthy, scope, sig: (rec_id_i32, r_bytes, s_bytes) }
+		FollowSchema { id, is_trustworthy, scope, sig: (rec_id_i32, r_bytes, s_bytes) }
 	}
 }
 
 impl Validation for FollowSchema {
-	fn validate(&self) -> Result<(PublicKey, bool), AttTrError> {
+	fn validate(&self) -> Result<(PublicKey, Did, bool), AttTrError> {
```

Comment:
Why return the signature verification result as part of an `Ok` result?  Wouldn't it make more sense to report verification failure as an error?